processor/k8sprocessor/kube/kube.go

```diff
@@ -142,3 +138,8 @@ type Association struct {
 	From string
 	Name string
 }
+
+// ExcludePods represent a list of Pods to ignore
+type ExcludePods struct {
+	Regex []*regexp.Regexp
```

Comment:
Add a field called `Pods` which will contain a regexp field called `Name` created from the name provided in config.